pgrx-examples/decoder/src/bin/pgrx_embed.rs

```diff
@@ -0,0 +1 @@
+::pgrx::pgrx_embed!();
```

Comment:
nit but I just don't like GitHub whining at me about it.
```suggestion
::pgrx::pgrx_embed!();

```